src/MiniExcel/DbList.cs

```diff
@@ -0,0 +1,176 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data.SQLite;
+using System.IO;
+using System.Text;
+
+namespace MiniExcelLibs
+{
+    public class DbList : IList<string>, IDisposable

```

Comment:
Why would you want to implement `IList<string>` interface? this class seems like SQLite connector from your code.

Reply:
Because sharedstring is truly List<T>, it seems like have some other property, but now, just ignore and use string content, so i just use List<string>
![image](https://user-images.githubusercontent.com/11787429/160224666-e300c211-c036-44ca-b56e-8159bbda6c2b.png)

Reply:
But this class operates with SQLite lite instead of a collection, isn't? 

Your screenshot doesn't seem to be related with  `DBList` implementing  `IList<T>` interface.

Reply:
currently get sharedstrings by index, in this line
https://github.com/MiniExcel/MiniExcel/blob/bbb060147964b5bf3fc14abd0db74d2f27364fe9/src/MiniExcel/OpenXml/ExcelOpenXmlSheetReader.cs#L687
this is why DbList inherit `IList<string>`